test/unit/getPastEventsFromHub.test.ts

```diff
@@ -0,0 +1,395 @@
+import type { RelayHub } from '@rsksmart/rif-relay-contracts';
+import { expect } from 'chai';
+import config from 'config';
+import type { providers } from 'ethers';
+import type { ServerConfigParams } from 'src';
+import {
+  getLogFilters,
+  getTopicsFromEvents,
+  performLogRequests,
+  splitRange,
+} from 'src/getPastEventsForHub';
+
+describe('getPastEventsFromHub', function () {
```

Comment:
what about the unit test for the main function `getPastEventsForHub`.

Reply:
I preferred to test its internals rather than adding a test to check that some functions are called. Do you think we need another one for the main function?

Reply:
we discussed about it.